src/templates/layout/base.twig

```diff
@@ -27,6 +27,20 @@
                         <li><a href="{{ url('watch.list') }}">Watch Functions</a></li>
                         <li><a href="{{ url('waterfall.list') }}">Waterfall</a></li>
                     </ul>
+                    {% set sites = sites() %}
```

Comment:
You still need to render it in Twig, I would leave the extension as is, having cleaned up the globals from there.